File: Java/src/br/com/dominio/s06/Exe09.java
package br.com.dominio.s06;

import java.util.Scanner;

public class Exe09 {

	public static void main(String[] args) {
		// variaveis
		float indice;
		Scanner teclado = new Scanner(System.in);
		
		//entradas
		System.out.println("Informe o indice de poluição: ");
		indice = teclado.nextFloat();
		
		//processamento
		if (indice < 0.3){
			System.out.println("Indice de puluição aceitavél. ");
		}else if(indice >= 0.3 && indice < 0.4) {
			System.out.println("ATENÇÃO! Indústrias do 1º grupo devem suspender as atividades. ");
		}else if(indice >= 0.4 && indice < 0.5) {
			System.out.println("ATENÇÃO! Indústrias do 1º e 2º grupo devem suspender as atividades. ");
		}else if(indice >= 0.5) {
			System.out.println("ATENÇÃO! Todos os Grupos devem suspender as atividades. ");
		}
		teclado.close();
	}

}

File: Python/Seção06/Exe09.py
#Entradas
indice = float(input("Insira o índice de poluição: "))
#processamento/saida
if indice >= 0.3 and indice < 0.4:
    print("Grupo 1 suspender as atividades")
if indice >= 0.4 and indice < 0.5: #pode se utilizar elif
    print("Grupo 1 e grupo 2 suspender as atividades")
if indice >= 0.5:
    print("Todos os grupos suspender as atividades")
if indice < 0.3:
    print("Niveís aceitáveis")